src/overlays/actors/ovl_En_Zos/z_en_zos.c

```diff
@@ -716,15 +743,15 @@ void EnZos_Draw(Actor* thisx, PlayState* play) {
 
     gfx = POLY_OPA_DISP;
 
-    gSPSegment(&gfx[0], 0x08, Lib_SegmentedToVirtual(D_80BBC75C[this->unk_2AC]));
+    gSPSegment(&gfx[0], 0x08, Lib_SegmentedToVirtual(sEyeTextures[this->unk_2AC]));
```

Comment:
```suggestion
    gSPSegment(&gfx[0], 0x08, Lib_SegmentedToVirtual(sEyeTextures[this->eyeIndex]));
```
Could do something like this, but not really necessary for now. Just making a note that it's wildly inconsistent through the repo (I'm seeing `eyeIndex`, `eyeTexIndex`, `eyeTextureIndex`, `eyeState`, `blinkState`, `texIndex`)

Reply:
Oh yeah, it's crazy inconsistent at the moment. I was sort of holding off on this (since I was documenting the *object* not the *actor*), but why not, you know?